icons/semaphoreci.svg

```diff
@@ -1 +1 @@
-<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Semaphore CI</title><path d="M21.864 5.266a.595.595 0 01.61.257s.467.76.744 1.365c.275.604.542 1.453.542 1.453a.592.592 0 01-.208.628l-1.505 1.073a.757.757 0 00-.284.646s.084.573.096 1.066c.01.485-.04 1.045-.04 1.045a.752.752 0 00.316.628l1.561.993c.192.122.3.398.24.618 0 0-.222.862-.467 1.48-.242.62-.668 1.4-.668 1.4a.592.592 0 01-.595.287l-1.819-.337a.76.76 0 00-.66.245s-.381.489-.754.861c-.414.415-.997.857-.997.857a.76.76 0 00-.254.658l.306 1.824a.592.592 0 01-.298.59s-.79.412-1.414.645c-.623.233-1.488.44-1.488.44a.592.592 0 01-.61-.251l-.967-1.576a.756.756 0 00-.624-.328s-.496.04-.949.025c-.466-.015-.986-.09-.986-.09a.754.754 0 00-.646.281L8.97 23.555a.593.593 0 01-.628.207s-.85-.267-1.456-.543c-.605-.276-1.362-.74-1.362-.74a.592.592 0 01-.256-.61l.385-1.599c.08-.331.338-.796.579-1.037l3.98-3.993a3.7 3.7 0 005.026-5.025l3.988-3.989c.241-.24.705-.5 1.036-.579zM2.134 18.733a.595.595 0 01-.61-.257s-.467-.758-.743-1.364c-.276-.605-.543-1.453-.543-1.453a.592.592 0 01.207-.628l1.506-1.073a.757.757 0 00.283-.646s-.082-.573-.095-1.066c-.01-.485.04-1.046.04-1.046a.752.752 0 00-.316-.628L.303 9.58a.592.592 0 01-.241-.617S.285 8.1.528 7.48c.244-.617.67-1.4.67-1.4a.592.592 0 01.594-.286l1.819.337a.76.76 0 00.661-.244s.38-.489.753-.862c.414-.414.997-.855.997-.855a.76.76 0 00.254-.66L5.97 1.688a.592.592 0 01.298-.591S7.058.685 7.682.45C8.305.219 9.17.012 9.17.012a.592.592 0 01.612.25l.966 1.578a.756.756 0 00.624.326s.496-.039.949-.024c.466.015.986.091.986.091a.754.754 0 00.646-.283L15.028.445a.593.593 0 01.628-.208s.85.268 1.456.544c.605.275 1.362.74 1.362.74a.593.593 0 01.256.61l-.385 1.599a2.59 2.59 0 01-.578 1.036L13.786 8.76a3.7 3.7 0 00-5.026 5.026l-3.988 3.987c-.24.242-.705.5-1.036.58z"/></svg>
\ No newline at end of file
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Semaphore CI</title><path d="m21.50314 13.2549-4.15781-4.22828a3.03814 3.03814 0 0 0-4.3591 0L9.6943 12.374a1.20701 1.20701 0 0 1-1.7213 0l-1.63096-1.6587 4.1578-4.22866a6.53247 6.53247 0 0 1 9.34234 0L24 10.71531zM8.82879 19.47925a6.52947 6.52947 0 0 1-4.67098-1.9657L0 13.295l2.48674-2.52872 4.15744 4.21816a3.05613 3.05613 0 0 0 4.3591 0l3.29191-3.34814a1.20701 1.20701 0 0 1 1.7213 0l1.63097 1.6587-4.14732 4.22866a6.5186 6.5186 0 0 1-4.67135 1.95558z" /></svg>
```

Comment:
Thank you @jorgeamadosoria! Looking good now!

Out of curiosity, I gave this a try. I think I made it work with a precision of 3. It gave me the following path:
```
M14.43 4.563a6.533 6.533 0 00-3.93 1.923L6.34 10.715l1.632 1.66a1.207 1.207 0 001.722 0l3.292-3.349a3.038 3.038 0 014.36 0l4.156 4.229L24 10.715l-4.158-4.229a6.533 6.533 0 00-5.412-1.923zM2.487 10.766L0 13.294l4.158 4.219a6.53 6.53 0 004.67 1.966 6.519 6.519 0 004.672-1.956l4.147-4.229-1.63-1.658a1.207 1.207 0 00-1.722 0l-3.292 3.349a3.056 3.056 0 01-4.36 0z
```

How does it look on your side if you do the comparison?

Reply:
![image](https://user-images.githubusercontent.com/15719479/130109833-e1f7b2e1-7b17-4276-8ef7-00fcb8ce42fa.png)

Still slightly different on my inkscape. Is that what you see?

Reply:
![image](https://user-images.githubusercontent.com/15719479/130110599-06bd01fe-b53d-4741-b032-30cd3e9171ab.png)

I downloaded the source again to be sure, and United the path rather than combining them. The result is not perfect, but it's the same I see with the Walmart icon: the difference is only in the outline, like the icon after SVGO is only a border length wider than the source.

Reply:
This one is harder than it looks, sorry for this 😅 
There is a weird behavior while ungrouping the path working on this one. 

Can you commit this new version? I will check it out!

Reply:
We just need to keep trying, I suppose.
So let's :)

Reply:
I think the last comparison was good, right? if there are no other issues, I think this is good to merge.

Reply:
When possible, we try to keep a lower precision. I'm quite sure this icon does not require a precision as high as 5. This is why I proposed a path above.

Lets keep it this way for now since it is a perfect match! Merging it now. Thank you for your work @jorgeamadosoria!